### ISSUE TITLE
Shader Errors with Unity 2017.4.2f2 (macOS Build)

### ISSUE THREAD

**Issue by HulloImJay** [wrld3d/unity-api#82]
In Unity 2017.4.2f2 I get the following shader compile errors when attempting to build for macOS.

```
Shader error in 'Wrld/StandardWithVertexColor': 'FragmentSetup': cannot convert from 'half4[3]' to 'float4[3]' at Assets/Wrld/Shaders/UnityVC.cginc(154) (on glcore)

Compiling Vertex program with DIRECTIONAL LIGHTPROBE_SH _ALPHABLEND_ON _VERTEXCOLOR_OFF
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_RGBM_ENCODING
```

```
Shader error in 'Wrld/Stencil/StencilledGeometry': 'FragmentSetup': cannot convert from 'half4[3]' to 'float4[3]' at Assets/Wrld/Shaders/UnityVC.cginc(154) (on glcore)

Compiling Vertex program with DIRECTIONAL SHADOWS_SCREEN LIGHTPROBE_SH VERTEXLIGHT_ON _VERTEXCOLOR_OFF
Platform defines: UNITY_ENABLE_REFLECTION_BUFFERS UNITY_USE_DITHER_MASK_FOR_ALPHABLENDED_SHADOWS UNITY_PBS_USE_BRDF1 UNITY_SPECCUBE_BOX_PROJECTION UNITY_SPECCUBE_BLENDING UNITY_ENABLE_DETAIL_NORMALMAP SHADER_API_DESKTOP UNITY_COLORSPACE_GAMMA UNITY_LIGHT_PROBE_PROXY_VOLUME UNITY_LIGHTMAP_RGBM_ENCODING
```

Wrld Unity plugin v0.6.918

**Comment by tommymacdonald** [comment]
Hi,

This issue has been tracked internally and has already been fixed. The fix should become available in our next WRLD Unity SDK release, however it is currently available in our bleeding-edge snapshot build:

http://cdn1.wrld3d.com/unity/wrld.unity.sdk-SNAPSHOT.zip

The accompanying documentation is available here:

https://docs.wrld3d.com/unity-SNAPSHOT/latest/docs/api/

Please note this version may be less stable than our Asset Store releases.

Kind regards,
Thomas

**Comment by HulloImJay** [comment]
Awesome. Thanks!

**Comment by ghost** [comment]
Fixed in v0.8
